README.md

```diff
@@ -138,6 +140,10 @@ spec:
 
 # Changelog
 
+## 3.0.0
```

Comment:
Could you please set the actual version on the top of readme. It is mean that the current content updated to the specified version